.travis.yml

```diff
@@ -3,12 +3,14 @@ node_js:
   - "stable"
 cache:
   directories:
-  - node_modules
+  - frontend/node_modules
   - backend/node_modules
 script:
+  - cd frontend
```

Comment:
I don't think we need a script for this

Reply:
According to https://travis-ci.com/github/rhopensourcecontest/osc-app/builds/175371131 we do need it.